tests/cli/test_rasa_data.py

```diff
@@ -175,6 +175,8 @@ def test_data_validate_stories_with_max_history_zero(monkeypatch: MonkeyPatch):
             "data/test_moodbot/data",
             "--max-history",
             0,
+            "--config",
+            "data/test_moodbot/config.yml",
```

Comment:
CLI command uses default config path if not provided (`config.yml` in project root). That file does not exist in test env.

`RasaFileImporter` returns empty dict if config file does not exist. `TrainingDataImporter.load_config` fails hard in the same case.